Nos/Models/ReportCategory.swift

```diff
@@ -89,7 +89,13 @@ enum ReportCategoryType {
     static let nsfw = ReportCategory(
         name: .moderation.nsfw,
         code: "NW",
-        nip56Code: .other
+        nip56Code: .nudity
```

Comment:
While NSFW content may contain nudity, it's not the only option. [According to Wikipedia](https://en.wikipedia.org/wiki/Not_safe_for_work):

> The marked content may contain [graphic violence](https://en.wikipedia.org/wiki/Graphic_violence), [pornography](https://en.wikipedia.org/wiki/Pornography), [profanity](https://en.wikipedia.org/wiki/Profanity), [nudity](https://en.wikipedia.org/wiki/Nudity), [slurs](https://en.wikipedia.org/wiki/Pejorative) or other potentially disturbing subject matter.

I don't know how NIP-56 is typically used, so maybe `.nudity` is better than `.other` for NSFW, but I want to be sure we use the best option here.

Reply:
@setch-l what about changing the NSFW acronym to something else?

Reply:
🤔 why are we bending ourselves around what is in NIP-56? We know it doesn't work for us, that's why we have made our own categories. So let's keep NSFW as a category, it's very broadly useful, and continue mapping it to `other` for clients that only support NIP-56?

Reply:
If we use `other` we need to get back to use NIP-69 to get the specifics in addition to NIP-56 as we were doing before these changes. If we do that, then the buttons in the Slack channel should be NIP-69 categories, not NIP-56. Not all of what NIP-69 offers but only the same subset available through the app.

Reply:
I see. I've read the ticket comments now. I didn't realize we were completely overhauling the categories. This PR is still publishing events with NIP-69 labels, right? Are you planning to remove those in another PR?

Reply:
Not anymore, this was covered in https://github.com/planetary-social/nos/pull/1289/commits/cd865bf7ae48f9d099f9ef341d8319ab15a48775